web/src/containers/ApdExport.js

```diff
@@ -0,0 +1,83 @@
+import { Button } from '@cmsgov/design-system-core';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Waypoint from './ConnectedWaypoint';
+import { printApd } from '../actions/print';
+import { Section } from '../components/Section';
+import { FileDownload } from '../components/Icons';
+
+const Bold = ({ children }) => (
+  <span className="ds-u-font-weight--bold">{children}</span>
+);
+Bold.propTypes = { children: PropTypes.node.isRequired };
```

Comment:
Made these as helpers rather than write out those long class names over and over. 😬